frontend/src/container/NewWidget/RightContainer/Threshold/Threshold.tsx

```diff
@@ -197,147 +199,165 @@ function Threshold({
 							)}
+						</div>
+					)}
+					{(selectedGraph === PANEL_TYPES.VALUE ||
+						selectedGraph === PANEL_TYPES.TABLE) && (
+						<div className="value-table-alerts">
+							<Typography.Text className="typography">
+								If value {selectedGraph === PANEL_TYPES.TABLE ? 'in' : 'is'}
+							</Typography.Text>
 							{isEditMode ? (
-								<Select
-									style={{ minWidth: '200px', backgroundColor }}
-									bordered={!isDarkMode}
-									defaultValue={unit}
-									options={unitOptions}
-									onChange={handleUnitChange}
-									showSearch
-								/>
+								<div>
+									{selectedGraph === PANEL_TYPES.TABLE && (
+										<Space style={wrapStyle}>
+											<Select
+												defaultValue={tableSelectedOption}
+												options={tableOptions}
+												bordered={!isDarkMode}
+												showSearch
+												onChange={handleTableOptionsChange}
+												rootClassName="operator-input-root"
+												className="operator-input"
+											/>
+											<Typography.Text className="typography">is</Typography.Text>
+										</Space>
+									)}
+									<Select
+										defaultValue={operator}
+										options={operatorOptions}
+										onChange={handleOperatorChange}
+										bordered={!isDarkMode}
+										style={{ marginLeft: '10px' }}
+										rootClassName="operator-input-root"
+										className="operator-input"
+									/>
+								</div>
 							) : (
-								<ShowCaseValue width="200px" value={unit} />
+								<div>
+									{selectedGraph === PANEL_TYPES.TABLE && (
+										<Space>
+											<ShowCaseValue
+												value={tableSelectedOption}
+												className="typography-preview"
+											/>
+											<Typography.Text
+												className="typography"
+												style={{ marginRight: '10px' }}
+											>
+												is
+											</Typography.Text>
+										</Space>
+									)}
+									<ShowCaseValue
+										width="50px"
+										value={operator}
+										className="typography-preview"
+									/>
+								</div>
 							)}
-						</Space>
-					</div>
-					<div>
-						<Space direction="vertical">
-							<Typography.Text>Show with</Typography.Text>
-							<Space style={wrapStyle}>
-								{isEditMode ? (
-									<>
-										<ColorSelector setColor={setColor} thresholdColor={color} />
-										<Select
-											style={{ minWidth: '100px', backgroundColor }}
-											defaultValue={format}
-											options={showAsOptions}
-											onChange={handlerFormatChange}
-											bordered={!isDarkMode}
-										/>
-									</>
-								) : (
-									<>
-										<ShowCaseValue width="120px" value={<CustomColor color={color} />} />
-										<ShowCaseValue width="100px" value={format} />
-									</>
-								)}
-							</Space>
-						</Space>
-					</div>
+						</div>
+					)}
 				</div>
-			</Card>
+				<div className="threshold-units-selector">
```

Comment:
Here, this could be refactored with fewer isEditMode condition

Reply:
will do later